server/@types/models/Referral.ts

```diff
@@ -1,6 +1,12 @@
-export type Referral = {
+type Referral = {
   id: string // eslint-disable-next-line @typescript-eslint/member-ordering
   offeringId: string
   prisonNumber: string
   referrerId: string
 }
+
+type ReferralIdWrapper = {
```

Comment:
Maybe `CreatedReferralResponse`? Might make it a bit more obvious what its use case is

Reply:
Yeah, that's a bit less nonchalant!